pom.xml

```diff
@@ -2347,8 +2347,12 @@
                 <!-- match 9+ -->
             </activation>
             <properties>
-                <modular.jdk.args>--add-exports=java.base/sun.nio.ch=ALL-UNNAMED
-                    --add-modules=java.se</modular.jdk.args>
+                <modular.jdk.args>
+                    --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
+                    --add-modules=java.se
+                    --add-opens=java.base/java.util=ALL-UNNAMED
```

Comment:
@ropalka When doing your work for #4595, did you not see a need for this --add-opens? 

I confess I brought this over because it was needed in full WF to run the subsystem tests. But that's not a good enough reason to bring it over to core. For any of these we should have a clear understanding of why we add them in the relevant spot. (And probably add a code comment.)

Reply:
I noticed it in WildFly code base @bstansberry https://github.com/wildfly/wildfly/pull/14304/commits/b3706a5989381175b042977b4ee61092ca0b4858